waku/persistence/store.go

```diff
@@ -410,6 +410,10 @@ func (d *DBStore) prepareQuerySQL(query *pb.HistoryQuery) (string, []interface{}
 	paramCnt++
 
 	sqlQuery += fmt.Sprintf("LIMIT $%d", paramCnt)
+	// pageSize
+	pageSize := query.PagingInfo.PageSize + 1
+	parameters = append(parameters, pageSize)
+	//
```

Comment:
The `//` in 416 and 441 could be removed as they do not contain any comment at all